ethexe/contracts/src/Router.sol

```diff
@@ -297,7 +298,9 @@ contract Router is IRouter, OwnableUpgradeable, ReentrancyGuardTransient {
         require(router.codes[codeId] == CodeState.Validated, "code must be validated before program creation");
 
         uint128 baseFeeValue = baseFee();
-        uint128 executableBalance = baseFeeValue * 10;
+
+        // By default get 10 WVara for executable balance.
```

Comment:
Not important. Better to set it as immutable contract variable with documentation (this comment)